mlrun/config.py

```diff
@@ -1040,6 +1040,14 @@ def get_default_function_pod_requirement_resources(
             resource_requirement.pop(gpu)
         return resource_requirement
 
+    def force_ssl_redirect(self):
```

Comment:
We can because it is indeed very specific, however it is only used in Application runtime, not for every api gateway by default (this can be enabled using `api_gateway.force_ssl_redirect()` manually).
Do you think we should add this config check on every api gateway creation?